packages/paste-core/components/modal/src/ModalHeading.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Heading, HeadingProps} from '@twilio-paste/heading';
+
+export interface ModalHeadingProps extends Omit<HeadingProps, 'marginBottom' | 'variant' | 'as'> {
+  children: NonNullable<React.ReactNode>;
+  as?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+}
+const ModalHeading: React.FC<ModalHeadingProps> = ({children, ...props}) => {
+  return (
+    <Heading as="h2" {...props} marginBottom="space0" variant="heading30">
```

Comment:
Should this `as` be a default prop? It looks like you're setting it as an `h3` in the stories.

Reply:
Yeah, we could do I suppose.

Reply:
Updated with a default prop instead